id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/IdMinterIntegrationTest.scala

```diff
@@ -14,21 +15,27 @@ import uk.ac.wellcome.models.{
   UnifiedItem
 }
 import uk.ac.wellcome.platform.idminter.modules.IdMinterModule
-import uk.ac.wellcome.test.utils.IntegrationTestBase
+import uk.ac.wellcome.test.utils.{DynamoDBLocal, SNSLocal, SQSLocal}
 import uk.ac.wellcome.utils.JsonUtil
 
 class IdMinterIntegrationTest
-    extends IntegrationTestBase
+    extends IntegrationTest
+    with SQSLocal
+    with DynamoDBLocal
+    with SNSLocal
     with Eventually
     with IntegrationPatience {
 
+  override def topicName: String = "ingester"
+  override def queueName: String = "id_minter"
```

Comment:
Let's prefix these with `test_` so we can't get confused with the prod instances.